### PR TITLE
main/git: upgrade to 2.22.0

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/git/APKBUILD [hunk 1]
```diff
@@ -10,17 +10,16 @@
...
 pkgdesc="Distributed version control system"
-url="https://www.git-scm.com"
+url="https://www.git-scm.com/"
 arch="all"
 license="GPL-2.0-or-later"
 depends=""
 # we need tcl and tk to be built before git due to git-gui and gitk
 makedepends="zlib-dev openssl-dev curl-dev expat-dev perl-dev python2-dev
 	pcre2-dev asciidoc xmlto perl-error tcl tk"
-options="!check" # Check is currently failing on t3900-i18n-commit.sh ISO-2022-JP.txt
```
Comment (Ikke) [inline main/git/APKBUILD]: Any reason you removed this?

Comment (andypost) [reply main/git/APKBUILD]: > >>> WARNING: git: APKBUILD does not run any tests!

Means it needs `check()`

Comment (Ikke) [reply main/git/APKBUILD]: Like the comment that got removed says, some checks are failing. We could perhaps see if we can exclude those specific checks though.

I've been working on getting all tests green for Alpine, but not there yet.

Comment (J0WI) [reply main/git/APKBUILD]: I forgot to readd this line. UTF-16 checks are not passing.
Changes of this PR have been applied in b7386174f378d261c2503b73aabebf079b27238a and 8778e17f073a8ba0c242d15f9c50c69918a66fc3.

**Pull request closed without merging**